e2e/rescript-v10-JSX4/src/css_js_test.res

```diff
@@ -0,0 +1,620 @@
+    expect([%css("background-repeat: repeat-x")])->toBe({"backgroundRepeat": "repeat-x"})
+    expect([%css("background-repeat: repeat-y")])->toBe({"backgroundRepeat": "repeat-y"})
+    expect([%css("background-repeat: repeat")])->toBe({"backgroundRepeat": "repeat"})
+    expect([%css("background-repeat: space")])->toBe({"backgroundRepeat": "space"})
+    expect([%css("background-repeat: round")])->toBe({"backgroundRepeat": "round"})
+    expect([%css("background-repeat: no-repeat")])->toBe({"backgroundRepeat": "no-repeat"})
+    expect([%css("background-repeat: inherit")])->toBe({"background-repeat": "inherit"})
+  })
+
+  test("test two values", _ => {
+    expect([%css("background-repeat: repeat space")])->toBe({"backgroundRepeat": "repeat space"})
+    expect([%css("background-repeat: repeat repeat")])->toBe({"backgroundRepeat": "repeat repeat"})
+    expect([%css("background-repeat: round space")])->toBe({"backgroundRepeat": "round space"})
+    expect([%css("background-repeat: no-repeat round")])->toBe({
+      "backgroundRepeat": "no-repeat round",
+    })
+  })
+})
+
+describe("backgroundImage", () =>
+  test("test values", _ => {
+    expect([%css("background-image: none")])->toBe({"backgroundImage": "none"})
+    expect([%css("background-image: url(x)")])->toBe({"backgroundImage": "url(x)"})
+    /* expect([%css("background-image: linear-gradient(5deg, #FF0000 10%)")])->toBe({
+      "backgroundImage": "linear-gradient(5deg, #FF0000 10%)",
+    })
+    expect([%css("background-image: repeating-linear-gradient(6rad, #000000 20%)")])->toBe({
+      "backgroundImage": "repeating-linear-gradient(6rad, #000000 20%)",
+    })
+    expect([%css("background-image: radial-gradient(#FFFF00 30%)")])->toBe({
+      "backgroundImage": "radial-gradient(#FFFF00 30%)",
+    })
+    expect([%css("background-image: repeating-radial-gradient(#FFFF00 30%)")])->toBe({
+      "backgroundImage": "repeating-radial-gradient(#FFFF00 30%)",
+     }) */
+  })
+)
+
+/* describe("Gradient background", () =>
+  test("test values", _ => {
+    expect([%css("background: linear-gradient(45deg, #FF0000 0, #0000FF 100%)")])->toBe({
+      "background": "linear-gradient(45deg, #FF0000 0, #0000FF 100%)",
+    })
+    expect([%css("background: repeating-linear-gradient(45deg, #FF0000 0, #0000FF 10px)")])->toBe({
+      "background": "repeating-linear-gradient(45deg, #FF0000 0, #0000FF 10px)",
+    })
+    expect([%css("background: radial-gradient(#FF0000 0, #0000FF 100%) ")])->toBe({
+      "background": "radial-gradient(#FF0000 0, #0000FF 100%)",
+    })
+    expect([
+      %css("background: repeating-radial-gradient(#FF0000 0, #0000FF calc(20% + 5px))"),
+    ])->toBe({
+      "background": "repeating-radial-gradient(#FF0000 0, #0000FF calc(20% + 5px))",
+    })
+    expect([%css("background: conic-gradient(from 45deg, #FF0000 0, #0000FF 100%)")])->toBe({
+      "background": "conic-gradient(from 45deg, #FF0000 0, #0000FF 100%)",
+    })
+  })
+) */
```

Comment:
Seems like there are some inconsistencies in the Gradient types

Reply:
Can you add this comment on the source?

Reply:
Yes, I can!